src/templates/app/_includes/financial_form.jsx

```diff
@@ -34,26 +34,24 @@ const Values = () => (
     </React.Fragment>
 );
 
-const FinancialForm = () => (
-    <Fieldset legend={it.L('Financial Information')}>
+const TradingExperienceForm = () => (
+    <Fieldset id='trading_experience_form' legend={it.L('Trading Experience')}>
         <SelectRow con={Experience} id='forex_trading_experience' label={it.L('Forex trading experience')} />
         <SelectRow con={Frequency}  id='forex_trading_frequency'  label={it.L('Forex trading frequency')} />
 
-        <SelectRow con={Experience} id='indices_trading_experience' label={it.L('Indices trading experience')} />
-        <SelectRow con={Frequency}  id='indices_trading_frequency'  label={it.L('Indices trading frequency')} />
-
-        <SelectRow con={Experience} id='commodities_trading_experience' label={it.L('Commodities trading experience')} />
-        <SelectRow con={Frequency}  id='commodities_trading_frequency'  label={it.L('Commodities trading frequency')} />
-
-        <SelectRow con={Experience} id='stocks_trading_experience' label={it.L('Stocks trading experience')} />
-        <SelectRow con={Frequency}  id='stocks_trading_frequency'  label={it.L('Stocks trading frequency')} />
+        <SelectRow con={Experience} id='other_derivatives_trading_experience' label={it.L('Binary options trading experience')} />
```

Comment:
I named these as `binary_options_trading_experience` and `binary_options_trading_frequency` in `(get|set)_financial_assessment`